src/openstack_mcp_server/tools/response/keystone.py

```diff
@@ -0,0 +1,8 @@
+from pydantic import BaseModel
+
+
+# NOTE: In openstacksdk, all of the fields are optional.
+# In this case, we are only using description field as optional.
+class Region(BaseModel):
+    id: str
+    description: str | None = ""
```

Comment:
Zero Value를  str로 줬으면 Optional 처리는 안해도 될 것 같아요
```suggestion
    description: str = ""
```